src/entrypoint.cpp

```diff
@@ -167,6 +189,21 @@ namespace aperf {
       })
       ->option_text(" ");
 
+    struct SubcommandData {
+    std::string metric_command;
+    std::string metric_name;
+    unsigned int frequency;
+    std::string regular_expression;
+    };
+
+    std::vector<std::string> metric_strs;
+    std::regex pattern_metric("");
+    app.add_option("-m,--metric", metric_strs, "Extra metric to sample  "
+                   "based on an external profiler/metric command. ")
+      ->option_text("c:\"METRIC_COMAND\",n:\"METRIC_NAME\",f:SAMPLELING_FREQUENCY ( must "
```

Comment:
I would move this explanation to the detailed description (i.e. the third argument of ```add_option()```) and only put something like "CONFIG" inside ```option_text()```.

Reply:
Done